app/breakpoints.js

```diff
@@ -0,0 +1,6 @@
+export default {
+    mobile:  '(max-width: 768px)',
+    tablet:  '(min-width: 769px) and (max-width: 992px)',
+    desktop: '(min-width: 993px) and (max-width: 1200px)',
+    jumbo:   '(min-width: 1201px)'
```

Comment:
I assume these correspond to bootstrap's `xs`, `sm`, etc? Maybe comment to that effect, so they're not just magic numbers.